UnityProject/Assets/Scripts/Player/PlayerMove.cs

```diff
@@ -128,8 +129,12 @@ public bool IsSwappable
 
 	[HideInInspector] public PlayerNetworkActions pna;
 
-	[HideInInspector] [SyncVar(hook = nameof(SyncRunSpeed))] public float RunSpeed;
-	[HideInInspector] [SyncVar(hook = nameof(SyncWalkSpeed))] public float WalkSpeed;
+	[HideInInspector] [SyncVar(hook = nameof(SyncRunSpeed))]
```

Comment:
This and other cosmetic changes were made automatically by JB Rider (I pressed the alignment buttons and forgot that they would apply to the entire file).

If it's not OK, tell me.

Reply:
This is fine.